src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/tests/test_batch_commands.py

```diff
@@ -571,15 +571,15 @@ def test_batch_get_model_attrs(self):
         attrs = list(self.command_job._get_attrs(models.ResourceFile, 'task.resource_files'))
         self.assertEqual(len(attrs), 3)
         attrs = list(self.command_job._get_attrs(models.JobManagerTask, 'job.job_manager_task'))
-        self.assertEqual(len(attrs), 10)
+        self.assertEqual(len(attrs), 11)
```

Comment:
Will these number changes when the test is run in a live environment? All the tests will be eventually run against the Azure service regularly. The assertions should be able to accommodate the variations in the test envirnoment.